client/components/jetpack/daily-backup-status/status-card/visible-days-limit.tsx

```diff
@@ -49,29 +49,29 @@ const useDisplayDate = ( date: MomentInput ) => {
 	}, [ date, gmtOffset, timezone ] );
 };
 
-const BeyondRetentionPeriod: React.FC< OwnProps > = ( { selectedDate } ) => {
+const VisibleDaysLimit: React.FC< OwnProps > = ( { selectedDate } ) => {
 	const translate = useTranslate();
 	const displayDate = useDisplayDate( selectedDate );
 
 	const siteId = useSelector( getSelectedSiteId ) as number;
-	const retentionDays = useSelector( ( state ) =>
-		getSiteActivityLogRetentionDays( state, siteId )
+	const visibleDays = useSelector( ( state ) =>
+		getActivityLogVisibleDays( state, siteId )
 	) as number;
 
 	const dispatch = useDispatch();
 	useEffect( () => {
-		dispatch( recordTracksEvent( 'calypso_backup_retentionlimit_view', { site_id: siteId } ) );
```

Comment:
Similar to the other Tracks events, we should consider if these are already in use somewhere.